packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -645,19 +647,18 @@ const VirtualListBaseFactory = (type) => {
 					this.virtualListPause = new Pause('VirtualList');
 					this.virtualListPause.pause();
 
-					const containerNode = this.uiRef.containerRef;
-
-					if (containerNode && containerNode.addEventListener) {
-						containerNode.addEventListener('keyup', this.resumeSpotlight);
+					if (document) {
```

Comment:
I think we're safe to access `document` in this code and in `resumeSpotlight`. IIRC, the only lifecycle methods that run during prerendering where `window` and `document` are undefined are the constructor and `componentWillMount` (which is going away). @JayCanuck can correct me if I've misspoke.

Reply:
For prerender support, `window`/`document` should be avoided in `constructor`, `getDerivedStateFromProps`, `componentWillMount` and `render`, yep.